dataset_description.json

```diff
@@ -1,5 +1,7 @@
 {
 	"Name": "data_axondeepseg_sem",
-	"BIDSVersion":"1.6.0 - BEP031 v0.0.4",
+	"BIDSVersion": "1.6.0 - BEP031 v0.0.5",
+	"Authors": "List of individuals who contributed to the creation/curation of the dataset.",
+	"HowToAcknowledge": "Text containing instructions on how researchers using this dataset should acknowledge the original authors. This field can also be used to define a publication that should be cited in publications that use the dataset.",
```

Comment:
@jcohenadad
The lines `Authors` and `HowToAcknowledge` contains the description from the BIDS-spec. Would you like to add them to the dataset? If yes, with what content?

Reply:
yes pls-- @alexfoias can you you please help-- i have zero time

Reply:
@jcohenadad, as discussed, I removed the `HowToAcknowledge` field and added the `Authors` list.
The list include all authors from the ADS paper, the people acknowledged in the paper for their help on acquiring the SEM data and labelling ground truth, Alex and myself.
The PR is ready for review.

Reply:
@mariehbourget is there anything else that I have to do for this PR?